eclass/nvidia-driver.eclass

```diff
@@ -129,42 +136,42 @@ nvidia-driver_get_mask() {
 
 	for nvidia_gpu in ${nvidia_gpus}; do
 		for drv in ${drv_71xx}; do
```

Comment:
Can't you use `has` here and below? Also, might be cleaner inline the vars or at least keep them cleaner to the actual conditions.